datalake-api/src/main/java/com/sequenceiq/sdx/api/model/SdxDatabaseServerResourceStatus.java

```diff
@@ -0,0 +1,8 @@
+package com.sequenceiq.sdx.api.model;
+
+public enum SdxDatabaseServerResourceStatus {
+
+    UNKNOWN,
```

Comment:
I seen the same domain class in case of Distrox should it be make sense to unify somehow?

Reply:
I think the domain classes of APIs should not depend on each other. Of course this will cause code duplication.
Is there any common agreement about this?

Reply:
there are common objects in the code aready, like distrox endpoint and sdx detail endpoint also giving back a stackResponse. I think if we can do it only once and use the same in both place then it reduce the possible mistakes in the future.